spec/tenPinSpec.js

```diff
@@ -0,0 +1,172 @@
+describe('TenPin', function() {
+
+  beforeEach(function(){
+    frame = new Frame();
+    tenP = new TenPin(Frame);
+    roll1 = new Roll;
+    roll2 = new Roll;
+    rollBonus = new Roll;
+  });
+
```

Comment:
You are defining real objects here, so spies are not really necessary, you can just test the actual instances.